test/AspNet.Security.OAuth.Providers.Tests/ExactOnline/bundle.json

```
@@ -0,0 +1,61 @@
+{
+  "$schema": "https://raw.githubusercontent.com/justeat/httpclient-interception/master/src/HttpClientInterception/Bundles/http-request-bundle-schema.json",
+  "items": [
+    {
+      "uri": "https://start.exactonline.nl/api/oauth2/token",
+      "method": "POST",
+      "contentFormat": "json",
+      "contentJson": {
+        "access_token": "secret-access-token",
+        "token_type": "Bearer",
+        "refresh_token": "secret-refresh-token",
+        "expires_in": "600"
+      }
+    },
+    {
+      "uri": "https://start.exactonline.nl/api/v1/current/Me",
+      "contentFormat": "json",
+      "contentJson": {
+        "d": {
+          "results": [
+            {
+              "__metadata": {
+                "uri": "https://start.exactonline.nl/api/v1/current/Me(guid'00000000-0000-0000-0000-000000000000')",
+                "type": "Exact.Web.Api.System.Me"
+              },
+              "CurrentDivision": 12345,
+              "DivisionCustomer": "00000000-0000-0000-0000-000000000001",
+              "DivisionCustomerCode": "          12345678",
```

Should this have all the leading spaces?

```suggestion
              "DivisionCustomerCode": "12345678",
```

---

Yes (unfortunately). These codes are defined as a 15-character numeric string, left-padded with spaces.